packages/theme/css/foundations/shadow.css

```diff
@@ -1,24 +1,23 @@
 .salt-theme[data-mode="light"] {
-  --salt-shadow-1-color: rgba(0, 0, 0, 0.1);
-  --salt-shadow-2-color: rgba(0, 0, 0, 0.1);
-  --salt-shadow-3-color: rgba(0, 0, 0, 0.15);
-  --salt-shadow-4-color: rgba(0, 0, 0, 0.2);
-  --salt-shadow-5-color: rgba(0, 0, 0, 0.3);
+  --salt-shadow-100-color: rgba(0, 0, 0, 0.1);
+  --salt-shadow-200-color: rgba(0, 0, 0, 0.1);
+  --salt-shadow-300-color: rgba(0, 0, 0, 0.15);
+  --salt-shadow-400-color: rgba(0, 0, 0, 0.2);
+  --salt-shadow-500-color: rgba(0, 0, 0, 0.3);
 }
 
 .salt-theme[data-mode="dark"] {
-  --salt-shadow-1-color: rgba(0, 0, 0, 0.5);
-  --salt-shadow-2-color: rgba(0, 0, 0, 0.5);
-  --salt-shadow-3-color: rgba(0, 0, 0, 0.55);
-  --salt-shadow-4-color: rgba(0, 0, 0, 0.55);
-  --salt-shadow-5-color: rgba(0, 0, 0, 0.65);
+  --salt-shadow-100-color: rgba(0, 0, 0, 0.5);
+  --salt-shadow-200-color: rgba(0, 0, 0, 0.5);
+  --salt-shadow-300-color: rgba(0, 0, 0, 0.55);
+  --salt-shadow-400-color: rgba(0, 0, 0, 0.55);
+  --salt-shadow-500-color: rgba(0, 0, 0, 0.65);
```

Comment:
worth asking if these opacities should come from opacity foundation. i think i asked before but since theme is being re-worked best to double checxk

Reply:
the tokens in shadow reference a very specific few opacity values that don't all exist in opacity foundations (in fact only 1 value out of 7 used is available in opacity foundations). It makes sense to me they have their own opacity values instead of available globally, but i've raised this with Paul just to confirm as well

Reply:
Quoting Paul
> This is a really good spot, but I think for now we shouldn't worry about this. Within Salt 2.0 theme I think there's room for us to relook at our shadows.